website/src/theme/DocCategoryGeneratedIndexPage/index.tsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import {useLayoutDoc} from '@docusaurus/theme-common';
+import Link from '@docusaurus/Link';
+import Translate from '@docusaurus/Translate';
+import DocCategoryGeneratedIndexPage from '@theme-original/DocCategoryGeneratedIndexPage';
+import type {Props} from '@theme/DocCategoryGeneratedIndexPage';
+
+import styles from './styles.module.css';
+
+export default function DocCategoryGeneratedIndexPageWrapper(
+  props: Props,
+): JSX.Element {
+  const docPath = useLayoutDoc('guides/docs/sidebar/items', undefined)?.path;
+  return (
+    <>
+      <DocCategoryGeneratedIndexPage {...props} />
+      <p className={styles.footerTip}>
```

Comment:
maybe create a separate comp just for this paragraph? would make the intent clearer and also avoid mixing different levels of abstractions